src/components/language-switcher.tsx

```diff
@@ -0,0 +1,39 @@
+import { Select, SelectProps } from '@chakra-ui/react'
+import { useRouter } from 'next/router'
+
+const languages = [
+  { label: '🇬🇧', value: 'en' },
+  { label: '🇩🇪', value: 'de' },
+]
```

Comment:
What if we have the whole names of the language instead instead of GB/DE? Also to me `EN` makes more sense since GB more reminds me of the country rather than the language.

Reply:
The short form here is a convention for localization. It's the most convienient way to work with it.